crates/rmf_site_format/src/asset_source.rs

```diff
@@ -33,6 +33,7 @@ pub enum AssetSource {
     Remote(String),
     Search(String),
     Package(String),
+    Ros(String),
```

Comment:
I wonder if there may be a better name for this asset type than `Ros`. There's nothing in its implementation or how it's used that associates it with ROS. If anything it looks like it's for loading in-memory serialized models that need to be parsed.

Maybe we should call this `Memory`?